pycloudlib/instance.py

```diff
@@ -311,13 +317,15 @@ def _ssh_connect(self):
                     username=self.username,
                     hostname=self.ip,
                     port=int(self.port),
+                    timeout=self.connect_timeout,
                     key_filename=self.key_pair.private_key_path,
                 )
                 self._ssh_client = client
                 return client
             except (ConnectionRefusedError, AuthenticationException,
                     BadHostKeyException, ConnectionResetError, SSHException,
                     OSError) as e:
+                self._log.warning("Connection error: %s", e)
```

Comment:
can we change this text so that it doesn't say error in a warning message (because we are retrying).

I had a previous suggestion that would work, though I think log.warning is reasonable instead of log.info.